zh_cn/news/_posts/2015-03-03-ruby-2-2-1-released.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: news_post
+title: "Ruby 2.2.1 发布"
+author: "hsbt"
+translator: "cassiuschen"
+date: 2015-03-03 03:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 2.2.1 正式发布了！
+该版本是 Ruby 2.2 系列的第一个补丁版本。
```

Comment:
This TENNY is important. Read more about it [here](https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/). :wink: